src/types/constants.ts

```diff
@@ -2,3 +2,5 @@ export enum TruncateMaxNumber {
   SELECTOR = 16,
   CLIENT_TASK_DESCRIPTION = 256,
 }
+
+export const signedUrlTtl = 60 * 60
```

Comment:
Is there any particular reasons we are writing `60*60` instead of directly writing this as `3600`?